### PR TITLE
fix(cdk8s): unable to obtain root node

### PR DESCRIPTION
**Pull request by Chriscbr**
Fixes https://github.com/winglang/wing/issues/6806

### CODE REVIEW

File: .github/workflows/cdk8s-release.yaml
Comment (MarkMcCulloh) [inline .github/workflows/cdk8s-release.yaml:1]: This file is marked as "generated" in the gitattributes, but it's only initially generated right?

File: cdk8s/examples/nodejs.main.w [hunk 1]
```diff
@@ -1,8 +1,9 @@
 bring "cdk8s-plus-27" as k8s;
+bring fs;
 
 // lets create a volume that contains our app.
 let appData = new k8s.ConfigMap();
-appData.addDirectory("./nodejs-app");
+appData.addDirectory(fs.join(@dirname, "./nodejs-app"));
```
Comment (MarkMcCulloh) [inline cdk8s/examples/nodejs.main.w:6]: nit:

```suggestion
appData.addDirectory(fs.join(@dirname, "nodejs-app"));
```

File: cdk8s/root.test.w
Comment (MarkMcCulloh) [inline cdk8s/root.test.w:1]: out of context this test seems strange imo, maybe it's worth linking to the issue

File: .github/workflows/cdk8s-release.yaml [hunk 2]
```diff
@@ -27,9 +27,10 @@ jobs:
       - name: Test
         run: |
           #!/bin/sh
-          cd examples
-          wing compile -t ../lib/index.js nodejs.main.w
-          wing compile -t ../lib/index.js ubuntu.main.w
+
+          wing compile -t ./lib/index.js examples/nodejs.main.w
```
Comment (MarkMcCulloh) [inline .github/workflows/cdk8s-release.yaml:31]: We could run the test.sh here instead, to avoid out-of-sync testing scenarios

**Review by MarkMcCulloh — APPROVED** [review]

File: .github/workflows/cdk8s-release.yaml (hunk 2 above)
Comment (Chriscbr) [reply .github/workflows/cdk8s-release.yaml:31]: Sure, let me try that

Comment (Chriscbr) [reply .github/workflows/cdk8s-release.yaml]: Done

**Pull request merged**